runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -328,13 +345,23 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 
+		if (config.screenshotKick())
+		{
+			if (chatMessage.startsWith("Your request to kick"))
```

Comment:
Combine these conditions
```suggestion
		if (config.screenshotKick() && chatMessage.startsWith("Your request to kick"))
```